### PR TITLE
docs(docs): document components available in MDX 

### PR DESCRIPTION
**Pull request by CollierCZ**
with an example page. Also includes typography and such. Other components should be added to the page once they're ready.

 Orbit.kiwi: https://orbit-docs-docs-mdx-examples.surge.sh
 Storybook: https://orbit-docs-mdx-examples.surge.sh

### CODE REVIEW

File: docs/src/mdx-components.tsx [hunk 1]
```diff
@@ -82,7 +91,14 @@ export const th = ({ children, align }: React.ThHTMLAttributes<HTMLTableHeaderCe
   </TableCell>
 );
 export const ul = ({ children }: React.HTMLAttributes<HTMLUListElement>) => (
-  <ul style={{ listStyle: "inside none disc" }}>{children}</ul>
+  <ul
```
Comment (mainframev) [inline docs/src/mdx-components.tsx:94]: can you also add spacing between components? it looks kinda sticky :)

**Review by mainframev — COMMENTED** [review]
LGTM :)

File: docs/src/pages/mdx-examples.mdx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+Code blocks
+
+```txt
+Sample text here...
+```
+
+Syntax highlighting
+
+```javascript
+var foo = function (bar) {
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/pages/mdx-examples.mdx:106]: That's what I get for copying examples without looking 🙈

File: docs/src/mdx-components.tsx (hunk 1 above)
Comment (CollierCZ) [reply docs/src/mdx-components.tsx:94]: Added some

**Review by mainframev — APPROVED** [review]

**Pull request merged**